src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -92,6 +92,7 @@ class Wedge3D {
   Direction<3> direction_of_wedge_{};
   double sphericity_of_other_surface_{
       std::numeric_limits<double>::signaling_NaN()};
+  bool with_tan_map_ = false;
```

Comment:
I think a better name would describe the outcome (equiangular vs. equidistant) rather than the mathematical form (I know SpEC is to blame ;)). Perhaps `with_equiangular_mapping`?

Regardless of final name choice, this option should be described in the Doxygen above.